### DISCUSSION TITLE
Perte de toute information: TypeError: 'bool' object is not callable

### DISCUSSION THREAD

**Discussion by Baloo42300** [davidrapan/ha-solarman#311] [Q&A] [answered]
Bonjour , depuis la mise a jour de ce soir j'ai perdu toute la remonté d'info. Jai ceci qui remonte de mon journal : 
Cette erreur provient d'une intégration personnalisée

Enregistreur: homeassistant
Source: custom_components/solarman/entity.py:64
intégration: Solarman (documentation, problèmes)
S'est produit pour la première fois: 21:44:24 (2 occurrences)
Dernier enregistrement: 21:44:25

Error doing job: Task exception was never retrieved (None)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 267, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 478, in _async_refresh
    self.async_update_listeners()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 177, in async_update_listeners
    update_callback()
    ~~~~~~~~~~~~~~~^^
  File "/config/custom_components/solarman/entity.py", line 69, in _handle_coordinator_update
    self.async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1023, in async_write_ha_state
    self._async_write_ha_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1148, in _async_write_ha_state
    self.__async_calculate_state()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1084, in __async_calculate_state
    available = self.available  # only call self.available once per update cycle
                ^^^^^^^^^^^^^^
  File "/config/custom_components/solarman/entity.py", line 64, in available
    return self.coordinator.last_update_success and self.coordinator.inverter.available()
                                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
TypeError: 'bool' object is not callable

Je ne sais pas trop ce que ça veut dire. Il y as t'il un moment de revenir a l'encienne version svp ? Merci pour vos réponse rapide 🙏🏾🙏🏾

**Comment by davidrapan** [comment, marked as answer]
Bonjour @Baloo42300, c'est corrigé dans la version plus récente.

**Reply by Baloo42300** [reply]
@davidrapan  comment l'installer ? Je n'arrive pas a changer de version , pour moi celle que j'avais installé était la dernière

**Reply by Baloo42300** [reply]
@davidrapan svp ? Je ne retrouve pas la dernière version mais je vois bien ici :
![Screenshot_2024-12-09-22-05-13-99_c3a231c25ed346e59462e84656a70e50.jpg](https://github.com/user-attachments/assets/4cc06cc8-f28b-46d9-bdde-bf236394036d)
 que je ne suis pas a jour. Je ne sais pas comment faire autrement que quand vous envoyez vous même la mise a jour en proposition

**Reply by davidrapan** [reply]
[v24.12.08_10](https://github.com/davidrapan/ha-solarman/releases/tag/v24.12.08_10) est la dernière version. Sur la page d'intégration de HACS Solarman, dans le coin supérieur droit, cliquez sur le menu à 3 points et cliquez sur Mettre à jour les informations. Elle devrait apparaître.

**Reply by davidrapan** [reply]
Oui, c'est ça.

**Reply by Baloo42300** [reply]
Merci beaucoup, tout est rentré dans l'ordre 😁😉🙏🏾
